packages/primitives/radio/meta.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { TComponentConfig } from 'autoprops'
+import { component, startWithType } from 'refun' // mapState,
+import { RadioInput, RadioGroup, TRadioInput, TRadioGroup } from './src'
+
+export const Component = component(
+  startWithType<TRadioInput & TRadioGroup>()
+)(({ initialValue, ...radioInputProps }) => {
+  return (
+    <RadioGroup
```

Comment:
should the group appear in sandbox?

Reply:
No, group is essentially just the `Context.Provider`, my initial idea was to keep the footprint as minimal as possible. The goal is allowing the consumer to:

```jsx
<RadioGroup initialValue="bla">
<Component1>
   <Component2>
       <RadioInput />
     </Component2>
  </ Component1>
   <Component3>
       <RadioInput />
   </Component3>
</RadioGroup>
```